src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue

```diff
@@ -417,21 +417,21 @@ onMounted(async () => {
                   :metricsuffix="'MSGS'"
                 />
                 <!--Queue Length-->
-                <div class="col-xs-12 no-side-padding graph-values">
+                <div class="no-side-padding graph-values">
                   <div class="queue-length-values">
-                    <div class="row">
+                    <div class="">
```

Comment:
Can we get rid of the class attribute here?

```suggestion
                    <div>
```

Alternatively, is the div providing anything here?